papers/whitepaper.md

```diff
@@ -0,0 +1,54 @@
+
+In order to empower the next generation applications in the intersection of Web3 and AI, both data and model privacy is of utmost importance. Through the use of TEEs, we will not only be able to provide verifiable AI compute, but also provide full data privacy to the end user. This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party. Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data economic revenue models, as well applications that can operate on the user's behalf without revealing anything about the user. 
+
+On another note, the development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.
+
+A decentralized platform such as Atoma Network will put forth combined efforts that will ultimately lead to a new era of use cases within decentralized applications, driving financial incentives for computing power to be partially decouple from centralized cloud providers. Smart contracts will not only deploy new financial products, but also be able to deploy and orchestrate compute resources, powering new decentralized protocols. 
```

Comment:
As a decentralized platform, Atoma Network will put forth combined efforts that will ultimately lead to a new era of use cases within decentralized applications through driving financial incentives for computing power to be partially decoupled from centralized cloud providers. Smart contracts will not only deploy new financial products, but also be able to deploy and orchestrate compute resources, powering new decentralized protocols.